pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -810,6 +813,7 @@ func (m *kubeGenericRuntimeManager) removeContainer(containerID string) error {
 	if err := m.removeContainerLog(containerID); err != nil {
 		return err
 	}
+	m.cpuManager.UnregisterContainer(containerID)
```

Comment:
Right now the error is discarded but that probably is not the right thing to do. In case unregistration fails we probably want to fail `removeContainer` so that it gets called again. Otherwise we could leak/strand CPUs.

CC @sjenning